### ISSUE TITLE
delta table add Time Travel support

### ISSUE THREAD

**Issue by alitrack** [roapi/roapi#165]
if possible add Time Travel support, deltalake supports.

**Comment by houqp** [comment]
Yes, delta-rs already supports time travel. The quickest way to add this is to add a dedicated query parameter to specify time or version for a table we want to query.
